include/zephyr/net/canbus.h

```diff
@@ -4,6 +4,10 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+/** @file
+ *  @brief CANBUS definitions.
```

Comment:
Only CAN is an abbreviation (for Controller Area Network), so the correct name is CAN bus. We also have the CAN peripheral driver and this header is only used for the Socket API, so I suggest the following change.

```suggestion
 *  @brief CAN bus socket API definitions.
```